fall18/p4/README.md

```diff
@@ -1,3 +1,308 @@
 # Project 4
 
-More details coming soon.
+This project is for all you movie enthusiasts out there! This week, we will be looking at data set about ~1,000 most popular movies on IMDB in the last 10 years. Each movie in the dataset has like cast, director, rating, genre, runtime, budget, etc.
+
+We will be working with **strings**, **loops**, **conditionals** and **command line arguments** in this project.
+
+To start, download the 4 files given below into your project directory.
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/test.py)
+* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/IMDB-Movie-Data.csv)
+
+Open up the IMDB-Movie-Data.csv file and get familiar with it! Notice how the movies are indexed starting from zero, how the 'genres' and 'cast' are listed as several actors separated by spaces. There's something strange about the column 'budget', can you tell what it is?
+
+**A reminder:** You should make your changes ONLY to the **main.py** file.
+DO NOT EDIT ANY OTHER FILES.
+
+After downloading all the files into the same directory try running the command below:
+
+```
+python test.py
+```
+
+and you should see the following: 
+```
+To be filled
+
+```
+
+# Introduction to Command Line inputs
+
+Important Note :  In this project we will be working on using commandline arguments to pass inputs to your program. Command-line arguments are specified while running the program file on the commandline. For example consider the following:
+
+```python
+python main.py upper "Steven Spielberg"
+``` 
+
+The above will pass the two arguments upper and "Steven Spielberg" to main.py via the command line and are called commandline arguments. We have written code inside the main.py file provided to you to read these arguments and provide to you for use in defining the functions for this project.
+In the code provided to you, the command line arguments are called command1 and command2. In the above example, command1 contains the string "upper" and command2 contains the string "Steven Spielberg".
+You will be enabling functionality for these command line arguments, so that you can use the functions you write directly from the command line!
+
```

Comment:
Note that the text we want command2 to contain is in **double quotes**. This is to ensure that "Steven Spielberg" is read as a single commandline argument and not two different ones. (Python separates command line arguments on the basis of intermediate spaces)